app/javascript/components/project/Project.vue

```diff
@@ -248,6 +255,17 @@ export default {
         })
         .catch(this.alertOrNotifyResponse);
     },
+    downloadExcelExport: function () {
+      axios
+        .get(`/projects/${this.project.id}/export/excel`)
+        .then((response) => {
+          FileDownload(
+            base64StringToBlob(response.data.replace(/(\r\n|\n|\r)/gm, ""), "application/xlsx"),
+            `${this.project.name}.xlsx`
+          );
+        })
+        .catch(this.alertOrNotifyResponse);
+    },
```

Comment:
It seems the download method used for downloading this versus the one used for downloading component exports are very different, could these be consolidated on a single download method?

Reply:
done